src/StoreApi/docs/cart.md

```diff
@@ -7,18 +7,14 @@
     -   [Error Response](#error-response)
 -   [Get Cart](#get-cart)
 -   [Add Item](#add-item)
+-   [Add Items](#add-items)
```

Comment:
Looks good to me, I think I would skip having this as a dedicated section because everything else points to an exact endpoint, maybe this could be a subsection of add-item? not a blocker, just a nitpick.

Reply:
Thanks for pointing this out, @senadir. I removed this section title and listed the change as a subsection.